composer.json

```
@@ -16,13 +16,13 @@
         }
     ],
     "require": {
-        "php": "^7.1 || ^7.3 || ^8.0",
-        "aws/aws-sdk-php": "^3.145",
-        "illuminate/support": "5.7.* || 5.8.* ||^6.0 || ^7.0 || ^8.0"
+        "php": "^8.0",
+        "aws/aws-sdk-php": "^3.219",
+        "illuminate/support": "^9.8"
```

Can you add back the previous version support? 

```suggestion
        "php": "^7.4 || ^8.0",
        "aws/aws-sdk-php": "^3.219",
        "illuminate/support": "^6.0 || ^7.0 || ^8.0 || ^9.8"
```